src/commonmark/en/content/developer/web-api/metadata.md

```diff
@@ -3947,15 +3947,16 @@ The contents of file resources are not directly accessible but are
 referenced from other objects (such as data values) to store binary
 content of virtually unlimited size.
 
-Creation of the file resource itself is done through either `/api/fileResources`
-or `/api/dataValues/file` as a multipart upload POST-request:
+To create a file resource without creating a corresponding data value,
```

Comment:
Assigning a file resource to a data value is only one reason for creating a file resource. I liked better the way I suggested because it talked about the first way as just creating a file resource in itself, not "without a corresponding data value". But I can accept your wording too.

Reply:
Yeah, I see what you mean.  Is my revised text better?  I also added a note about the version number.